index.html

```diff
@@ -19,7 +19,7 @@
 
       // the specification's short name, as in
       // http://www.w3.org/TR/short-name/
-      shortName: "vc-jwt",
+      shortName: "vc-jose-cose",
```

Comment:
This seems to suggest that the <https://github.com/w3c/vc-jwt> repo should be moved to <https://github.com/w3c/vc-jose-cose>?

Reply:
yep, we would want to do that, after the working group approves.

Reply:
Well... as long as this is only a change of the Repo, that is a piece of cake.

However, does it mean to change the short name of the specification as well? We _can_ do that, but it requires all kinds of complicated set ups at the W3C system team level, because they will have to set up redirections. Remember that no W3C URL should be a 404, ie, the `https://www.w3.org/TR/vc-jwt` should live forever in the form of a redirection. In essence, we would have two /TR URLs for the same document, even if the `vc-jwt` version would not be advertised.

Is this really worth the head ache and the trouble?

Reply:
I think renaming this is worth it. 

As far as _how_, perhaps it would be easier to create a new spec. Then in the text of the vc-jwt spec point to the new one, without redirecting?

Reply:
@iherman imo it is worth it for a lot of reasons, many of which have been documented in the most contentious PRs that we are not able to merge.

The current name no longer reflects working group consensus, based on resolutions we have passed, the discussions regarding mappings, and advancements that have been made outside the working group.

I leave it to the chairs to confirm, or clarify what I am summarizing.

@Sakurann @brentzundel

Reply:
It is my understanding that there is general agreement to re-name the VC-JWT spec to be more accurate. Done properly, this would require also changing the short name.
I see two options:
1. withdraw/retire the VC-JWT spec along with the short name `vc-jwt` and add text that explains where the new spec is.
2. rename the repository, introduce a new short name, and add a redirect to `vc-jwt`

from my perspective option 2 is better, but I am not the one who will need to deal with actually making any of this happen.

Reply:
I am good with option 2

Reply:
concurring with Brent's assessment.

Reply:
I propose that we (which may be me...) contact the webmaster to see what possible hurdles there are on option (2). If they are too important, we may have to fall back on option (1).

Do we have a consensus about trying (2)?

Reply:
@Sakurann @brentzundel please establish consensus on this topic.

Reply:
I'll put in on the agenda for next week's VCWG call